.editorconfig

```diff
@@ -2,6 +2,7 @@ root = true
 
 [*]
 indent_style = tab
+indent_size = 2
```

Comment:
Wondering if it'd actually be better to remove the indent rules entirely here. Since it's a boilerplate, they probably assume a bit too much.